plugins/BEdita/Core/src/Model/Table/UsersTable.php

```diff
@@ -392,6 +392,8 @@ protected function anonymizeUser(EntityInterface $entity)
         $entity->set('deleted', true);
         $entity->set('locked', true);
 
+        $this->ExternalAuth->deleteAll([$this->ExternalAuth->aliasField('user_id') => $entity->get('id')]);
```

Comment:
`Table::deleteAll()` doesn't invoke any callbacks on deleted entities. While it should be safe, I think we can assume that external auth methods will always be a small number for each user, thus I'd prefer to invoke `Table::delete()` on each row linked to the user being anonymized.